synapse/lib/cell.py

```diff
@@ -719,12 +734,16 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
     confdefs = {}  # type: ignore  # This should be a JSONSchema properties list for an object.
     confbase = {
         'cell:guid': {
-            'description': 'An optional hard-coded GUID to store as the permanent GUID for the cell.',
+            'description': 'An optional hard-coded GUID to store as the permanent GUID for the service.',
             'type': 'string',
+            'hideconf': True,
+            'hidecmdl': True,
```

Comment:
Adding hidecmdl in many places has the ability to break existing uses; its really overloaded to also mask precedence order for envars as well. We should probably change hidecmdl so it does exactly what it means to do, which is to hide things from being shown in the command line. I'll make a delta which makes this change.